relay.js

```diff
@@ -116,7 +115,8 @@ export function relayConnect(url, onNotice) {
     }
   }
 
-  const sub = async (channel, {cb, filter}) => {
+  const sub = async ({cb, filter}) => {
+    const channel = (await sha256(Math.random().toString())).reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '')
```

Comment:
There is a problem here. Only the first call should get a random channel ID. The following calls derived from this should reuse that ID.

Reply:
You see how each `sub()` call returns an object with `.sub()`? The idea is that calling that sub-`sub` function should make a new `REQ` call to the relay with the same previous `channel` id.

This is confusing.

Reply:
a25e6c4265374bf5448105d89439004a86132014